pkg/api/networkservice/mechanisms/memif/constants.go

```diff
@@ -31,4 +31,7 @@ const (
 
 	// SocketFileURL - file url for the memif socketfile
 	SocketFileURL = common.InodeURL
+
+	// SocketFileScheme - expected scheme of the SocketFileURL
+	SocketFileScheme = "file"
```

Comment:
Can we create a constructor for memif like in kernel https://github.com/networkservicemesh/api/blob/master/pkg/api/networkservice/mechanisms/kernel/helpers.go#L33?
It can look like 

```
func New(fileName string) *networkservice.Mechanism {
return Mechanism: &networkservice.Mechanism{
				Cls:  cls.LOCAL,
				Type: memif_mechanisms.MECHANISM,
				Parameters: map[string]string{
					memif_mechanisms.SocketFileURL: (&url.URL{Scheme: SocketFileScheme, Path: SocketFilename}).String(),
				},
			},
}
```

Reply:
Totally... but  I think that's thematically distinct from this PR :)